Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1226,6 +1227,50 @@ inline GridAutoFlow BuilderConverter::convertGridAutoFlow(BuilderState&, const C
     return autoFlow;
 }
 
+inline MasonryAutoFlow BuilderConverter::convertMasonryAutoFlow(BuilderState&, const CSSValue& value)
+{
+    auto* valueList = dynamicDowncast<CSSValueList>(value);
```

Comment:
Kind of curious why is this a ptr when the value is a reference?

Reply:
There really wasn't a reason. Changed to use downcast